services/functional-tests/src/test/java/org/infinispan/online/service/utils/CommandLines.java

```diff
@@ -0,0 +1,70 @@
+package org.infinispan.online.service.utils;
+
+import org.jboss.dmr.ModelNode;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+
+public class CommandLines {
```

Comment:
I don't like having two classes with the names `CommandLines` and `CommandLine`. Could we not just add the static methods from `CommandLines` to `CommandLine`?